src/main/java/socchat/Storage.java

```diff
@@ -0,0 +1,108 @@
+package socchat;
+
+import socchat.task.Task;
+import socchat.task.todo.*;
+import socchat.task.deadline.*;
+import socchat.task.event.*;
```

Comment:
Perhaps the imported classes should be listed explicitly as per the coding standard?